packages/frontend/src/components/verify-owner/VerifyOwner.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import IconCheck from '../../images/IconCheck';
+import CalimeroRoundedLogo from '../svg/CalimeroRoundedLogo';
+import NearRoundedLogo from '../svg/NearRoundedLogo';
+import SwapIcon from '../svg/WrapIcon';
+
+
+const StyledContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    
+    h3 {
+        color: #272729;
+        text-align: center;
+        font-weight: 700;
+        font-size: 24px;
+        line-height: 32px;
+    }
+
+    h2 { 
+        font-weight: 900;
+        font-size: 20px;
+        color: #272729;
+        text-align: center;
+        line-height: 130%;
+    }
+
+    > .logos {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        margin: 32px 0 32px 0;
+    }
+
+    .swap {
+        svg{
+            transform: rotate(90deg);
+            width: 24px;
+            height: 24px;
+            margin: 10px 4px 0 4px;
+        }
+    }
+
+    > .account {
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 20px;
+
+        &.signer {
+            border-top: none;
+            display: flex;
+            align-items: flex-end;
+            text-align: center;
+        }
+
+        &.permissions {
+            font-weight: 400;
+            font-size: 14px;
+            line-height: 20px;
+            text-align: justify;
+            text-color: #272729;
+            margin: 0 12px 12px 12px;
+            }
+        }
+
+    .check {
+        width: 32px;
+        height: 32px;
+        background: #D6EDFF;
+        border-radius: 50px;
+        padding-top: 3px;
+        margin-right: 15px;
+        svg {
+            width: 26px;
+            height: 26px;
+        }
+    }
+`;
+
+export default ({ accountId, appName }) => {
+    return (
+        <StyledContainer>
+            <h3><Translate id='verifyOwner.summary.title' /></h3>
+            <div className='logos'>
+                <CalimeroRoundedLogo />
```

Comment:
I suspect it won't always be Calimero requesting signature here? Might want to opt for something more generic like what's shown on the `/login` confirm screen (`SwapGraphic.js`):

<img width="545" alt="Screen Shot 2022-07-14 at 10 29 01 AM" src="https://user-images.githubusercontent.com/24921205/179006610-d970f4f3-1f48-4b96-9956-3ac7cbdc3d37.png">

Reply:
No, we actually want to be clear it is Calimero in this case.

Reply:
@xilosada @chefsale We should stick with a placeholder for now, so using SwapGraphic is the way to go until we come up with a solution to provide customizable icons there.